docs/content/docs/dev_guide/_index.md

```diff
@@ -10,9 +10,9 @@ weight: 5
 Welcome to the Open Service Mesh development guide!
 Thank you for joining us on a journey to build an SMI-native lightweight service mesh. The first of our [core principles](https://github.com/openservicemesh/osm#core-principles) is to create a system, which is "simple to understand and contribute to." We hope that you would find the source code easy to understand. If not - we invite you to help us fulfill this principle. There is no PR too small!
 
-To understand _what_ Open Service Mesh does - take it for a spin and kick the tires. Install it on your Kubernetes cluster by following [this guide](./example/README.md).
+To understand _what_ Open Service Mesh does - take it for a spin and kick the tires. Install it on your Kubernetes cluster by following [this guide](../install/).
 
-To get a deeper understanding of how OSM functions - take a look at the detailed [software design](../design/).
+To get a deeper understanding of how OSM functions - take a look at the detailed [software design](../design_concepts/).
```

Comment:
Similar to above, pointing to a dir will break on the docs site
```suggestion
To get a deeper understanding of how OSM functions - take a look at the detailed [software design](../design_concepts/index.md).
```
(sorry about this, we will make sure to document these docs site specific rules!)

Reply:
I believe `_index.md` is optional. Quick local test with `hugo serve` worked ok.

Reply:
Thanks, I'll follow up on this so we can update the repo readme for docs guidance